src/main/java/duke/Task.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+public abstract class Task {
```

Comment:
Needs header comments for every class and its non-trivial methods.
Since you are defining an abstract class, it is like a contract and becomes absolutely necessary to provide header comments here.